Octokit.Reactive/Clients/IObservableRepositoriesClient.cs

```diff
@@ -203,6 +203,10 @@ public interface IObservableRepositoriesClient
         /// <summary>
         /// Client for GitHub's Repository Statistics API
         /// </summary>
+        /// <summary>
```

Comment:
So this has added a 2nd `<summary>` section to the one that was already there.  I wasnt sure if this actually works or not.  Some brief testing I just did seems to indicate that the 2nd one is ignored... 

is this working for you?  If not, I think we need to append this info into the existing `<summary>` tags and not add duplicate ones

Reply:
Works On My Machine :tm:
![image](https://user-images.githubusercontent.com/1035315/55271598-27d43b80-52fb-11e9-82af-9acdcc64f67c.png)

This is in Rider though, I tested it in VS2017 and it does not show the second `<summary>` section.